src/components/common/slider/slider-handle.js

```diff
@@ -23,6 +23,7 @@ import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import styled from 'styled-components';
+import {getMouseDeltaX} from './slider-handle';
```

Comment:
I'm not sure I see this function defined anywhere in this file. Also, it seems to be importing from the same file.